toolbar-button/pyproject.toml

```
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["jupyter_packaging~=0.7.9", "jupyterlab>=3.0.0rc15,==3.*", "setuptools>=40.8.0", "wheel"]
```

We might want to use the 3.0 final release now that it's out:

```suggestion
requires = ["jupyter_packaging~=0.7.9", "jupyterlab~=3.0", "setuptools>=40.8.0", "wheel"]
```